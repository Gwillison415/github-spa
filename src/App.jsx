/* eslint-disable import/no-named-as-default */
import React from 'react';
import { Container } from 'semantic-ui-react';
import RepoList from './components/RepoList/RepoList';
import Login from './components/Login/Login';
import Organizations from './components/Organizations/Organizations';
import Dashboard from './components/Dashboard/Dashboard';
import './App.css';
import Issues from './components/Issues/Issues.jsx';
import PR_Table from './components/PR_Table/PR_Table.jsx'

const App = () => (
  <Container>
    <Login />
    <Organizations />
    <RepoList />
    <Dashboard />
  </Container>
);

export default App;
