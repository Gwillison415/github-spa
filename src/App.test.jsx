import React from 'react';
import toJson, { shallowToJson } from 'enzyme-to-json';
import { shallow, render, mount } from 'enzyme';
import App from './App.jsx';


describe('App component', () => {
  it('should render a component with props as specified ', () => {
    const component = shallow(
      <Provider>

        <App   />
      </Provider>
    );
    expect(toJson(component)).toMatchSnapshot();
  });
})
