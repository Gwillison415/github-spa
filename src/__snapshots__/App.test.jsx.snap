// Jest Snapshot v1, https://goo.gl/fbAQLP

<<<<<<< HEAD
exports[`App component should render a component with props as specified  1`] = `<Connect(App) />`;
=======
exports[`App component should render a component with props as specified  1`] = `
<Container>
  <Connect(Component) />
  <Connect(Organizations) />
  <Connect(RepoList) />
  <Connect(Component) />
</Container>
`;
>>>>>>> Fixing warning messages from tests
